test/planFinder.test.js

```diff
@@ -564,15 +564,19 @@ describe("PlanFinder tests - using dummy transformers", function() {
 });
 
 // Note: these tests should move to engine.test.js
-describe("E2E tests with Pipeline Engine", function() {
+describe("E2E tests with Pipeline Engine", function () {
+    const params = { skipMetadataExtraction: true };
+
     it("Runs a pipeline with one transformer and one refines the plan", async function() {
-        const pipeline = new Engine();
+        const pipeline = new Engine(params);
 
         pipeline.registerTransformer(new TransformerPNG());
         pipeline.registerTransformer(new TransformerGIF());
 
         const input = {
-            type: 'image/tiff'
+            type: 'image/tiff',
+            path: '/tmp/input.png',
```

Comment:
what if there is no path?
follow up qs: how do we know the input will have a path?